crates/bevy_transform/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub mod prelude {
 
     #[cfg(feature = "bevy-support")]
     #[doc(hidden)]
+    #[allow(deprecated)]
```

Comment:
Should we use the new `expect` from rust 1.81.0 ?

Reply:
Last I checked the MSRV hadn't been bumped yet. Has it?

Reply:
Feel free to bump it and use it here. Bevy's policy is "latest stable".